src/readout/ReadoutParserTestData.h

```diff
@@ -0,0 +1,86 @@
+/* Copyright (C) 2016-2018 European Spallation Source, ERIC. See LICENSE file */
+//===----------------------------------------------------------------------===//
+///
+/// \file
+///
+/// \brief Dataset for running unit tests - do not edit if unsure of what they do!
+///
+//===----------------------------------------------------------------------===//
+
+#include <cinttypes>
+#include <vector>
+
+// clang-format off
+
```

Comment:
Can we put a brief comment here that describes the LOKI data format? More practical than having always to look at the document.

Reply:
I'd rather not duplicate that. I'll add a \todo to adda URL to the description once 
the final format is agreed upon and documented somewhere.